GoogleDataLogger/Tests/Unit/Categories/GDLUploadCoordinator+Testing.h

```diff
@@ -14,19 +14,24 @@
  * limitations under the License.
  */
 
-#import "GDLTestCase.h"
-
 #import "GDLUploadCoordinator.h"
 
-@interface GDLUploaderTest : GDLTestCase
+NS_ASSUME_NONNULL_BEGIN
 
-@end
+@interface GDLUploadCoordinator (Testing)
+
+/** Resets the properties of the singleon, but does not reallocate a new singleton. */
+- (void)reset;
```

Comment:
With the singleton, how is running parallel tests handled without clobbering each other?